src/layout/NavigationButtons/NavigationButtonsComponent.tsx

```diff
@@ -154,6 +176,20 @@ function NavigationButtonsComponentInner({
         data-testid='NavigationButtons'
         className={classes.container}
       >
+        {showBackToComponentButton && (
+          <Button
+            disabled={isAnyProcessing}
+            isLoading={process === 'backToComponent'}
+            onClick={onClickBackToComponent}
+            // If we show any back button, "Next" should be secondary
+            variant={showBackToSummaryButton || showBackToComponentButton ? 'secondary' : 'primary'}
```

Comment:
Reads to me like this will always be true, as `showBackToComponentButton` is always true when this renders.